src/test/integration/uk/gov/hmcts/reform/sandl/snlevents/TestConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.sandl.snlevents;
+
+import org.springframework.context.annotation.Bean;
+import uk.gov.hmcts.reform.sandl.snlevents.testdata.builders.HearingPartBuilder;
+import uk.gov.hmcts.reform.sandl.snlevents.testdata.builders.SessionBuilder;
+import uk.gov.hmcts.reform.sandl.snlevents.testdata.builders.UpsertSessionBuilder;
+
+@org.springframework.boot.test.context.TestConfiguration
+public class TestConfiguration {
```

Comment:
I am pretty sure you wanted a SessionTestConfiguration or something connected by the package with session and hearings, not a global general one as there will be cases when below builders aren't necessary.